Package.swift

```diff
@@ -1,6 +1,15 @@
+// swift-tools-version:4.2
```

Comment:
we will likely not maintain this version if you want to remove

Reply:
Sweet, removed!